### DISCUSSION TITLE
control TransmitPower

### DISCUSSION THREAD

**Discussion by junyong-sim** [openthread/openthread#8114] [Q&A] [answered]
Hello,

I found the API `otPlatRadioSetTransmitPower
` in API reference. [https://openthread.io/reference/group/radio-config#otplatradiosettransmitpower]

1. After forming network, can I change transmit power with this API ?

2. Can you explain more about below note ? How to create the max power table for each channel ?
Note: The real transmit power will be no larger than the power specified in the max power table for the current channel.

**Comment by jwhui** [comment, marked as answer]
> After forming network, can I change transmit power with this API ?

These are platform APIs. The platform can choose to change the transmit power if it likes and Thread will continue to operate. However, note that Thread does evaluate link quality based on link margin, so changing the transmit power may also change the Thread topology.

> Can you explain more about below note ? How to create the max power table for each channel ? Note: The real transmit power will be no larger than the power specified in the max power table for the current channel.

The OpenThread radio platform API supports setting the maximum transmit power for a given channel [otPlatRadioSetChannelMaxTransmitPower()](https://openthread.io/reference/group/radio-operation#otplatradiosetchannelmaxtransmitpower). The note indicates that the configured maximum transmit power for the given channel will be respected.
